.gitignore

```diff
@@ -1,4 +1,171 @@
-src/*/obj
-src/*/bin
-GameData/*/Plugins
-*.userprefs
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
```

Comment:
I don't know where you got this .gitignore content from, but *ew*.  Most of this kind of crud belongs in site- or user-specific config, not in repo settings.  As a Linux & Monodevelop user, I don't need a bunch of .gitignore rules for VS crap and Windows' frigging thumbs.db.
I was thinking more along the lines of just "add src/lib".